client/components/telecom/telephony/alias/liveCalls/telecom-telephony-alias-configuration-liveCalls.html

```diff
@@ -158,61 +119,24 @@ <h2 class="widget-presentation-title" data-translate="telephony_alias_configurat
-               }">
-            <thead>
-                <tr>
-                    <th scope="col" data-translate="telephony_alias_configuration_mode_calls_agent"></th>
-                    <th scope="col" data-translate="telephony_alias_configuration_mode_calls_agent_status"></th>
-                    <th scope="col" data-translate="telephony_alias_configuration_mode_calls_agent_updated"></th>
-                </tr>
-            </thead>
-            <tbody data-ng-if="!$ctrl.isLoading">
-                <tr data-ng-repeat="agent in $ctrl.agentsStatusPaginated track by agent.agentId">
-                    <th scope="row" data-title="{{:: 'telephony_alias_configuration_mode_calls_agent' | translate }}">
-                        <span class="text-nowrap mr-3" data-ng-bind="agent.agentId"></span>
-                    </th>
-                    <td data-title="{{:: 'telephony_alias_configuration_mode_calls_agent_status' | translate }}">
-                        <span class="text-nowrap mr-3" data-ng-bind="'telephony_alias_configuration_mode_calls_agent_status_' + agent.status | translate"></span>
-                    </td>
-                    <td data-title="{{:: 'telephony_alias_configuration_mode_calls_agent_updated' | translate }}">
-                        <span data-translate="telephony_alias_configuration_mode_calls_agent_updated_ago"></span>
-                        &nbsp;
-                        <elapsed-time data-from="agent.lastStatusChange"></elapsed-time>
-                    </td>
-                </tr>
-            </tbody>
-            <tbody data-ng-if="$ctrl.isLoading">
-                <tr>
-                    <td colspan="3">
-                        <div class="text-center">
-                            <oui-spinner></oui-spinner>
-                        </div>
-                    </td>
-                </tr>
-            </tbody>
-            <tbody data-ng-if="!$ctrl.isLoading && !$ctrl.agentsStatus.length">
-                <tr>
-                    <td colspan="3"
-                        class="text-center"
-                        data-translate="telephony_alias_configuration_mode_agents_empty">
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-        <div class="mt-2 mb-3"
-             data-ng-show="!$ctrl.isLoading && $ctrl.agentsStatus.length"
-             data-pagination-front
-             data-items="$ctrl.agentsStatus"
-             data-paginated-items="$ctrl.agentsStatusPaginated"
-             data-current-page="currentPage"
-             data-nb-pages="nbPages"
-             data-items-per-page="itemsPerPage"
-             data-page-placeholder="{{ 'common_pagination_page' | translate: { currentPage:currentPage, nbPages:nbPages } }}"
-             data-item-per-page-placeholder="{{ 'common_pagination_display' | translate }}">
-        </div>
-    </div>
+
+    <oui-datagrid data-rows="$ctrl.agentsStatusPaginated">
```

Comment:
Another one bad named variable here. I assume that in the controller, `agentsStatusPaginated` does not contain paginated data...